tests/helpers/utils.ts

```diff
@@ -1,10 +1,11 @@
-import { type Options, execa } from 'execa';
-import fse from 'fs-extra';
 import fs from 'node:fs/promises';
 import os from 'node:os';
 import path from 'node:path';
 import { fileURLToPath } from 'node:url';
 
+import { execa, type Options } from 'execa';
```

Comment:
the lint meta-package groups all the `node` imports together, separate from external deps

Reply:
What exactly are you referring to as the "lint meta-package"?

Reply:
It'll be easier to see here (I just extracted this, in response to your comments): https://github.com/embroider-build/addon-blueprint/pull/175#issue-1852997419